### ISSUE TITLE
add option to delete ride from `My Rides`

### ISSUE THREAD

**Issue by foss-** [simra-project/simra-android#27]
On iOS rides can be deleted via swipe. On Android a swipe on a ride list view for `My Rides` does nothing. Long pressing does the same as short pressing: open ride.

Users should have a way to delete rides.

**Comment by selphiron** [comment]
Rides can be deleted before they are uploaded. For that, just tap on the trash bin icon.

**Comment by foss-** [comment]
Don't have a local ride currently to check. However wouldn't this still be an inconsistency with the iOS version? And why not allow users to delete uploaded rides (while keeping them on the server)?

**Comment by selphiron** [comment]
That is correct, we do not support swipe actions in "My Rides" on SimRa-Android. We do not allow to delete uploaded rides, because we don't have the connection between rides and users in the backend. This connection only stays within the app.

**Comment by foss-** [comment]
So that is the reason you don't allow to delete rides? Isn't the argument the same for iOS? So rather than filing a feature request on Android to allow swipe to delete, there should be rather a feature request to remove delete functionality from iOS variant?
